include/CL/cl2.hpp

```diff
@@ -1454,6 +1464,10 @@ CL_HPP_PARAM_NAME_INFO_1_2_DEPRECATED_IN_2_0_(CL_HPP_DECLARE_PARAM_TRAITS_)
 CL_HPP_PARAM_NAME_DEVICE_FISSION_(CL_HPP_DECLARE_PARAM_TRAITS_);
 #endif // CL_HPP_USE_CL_DEVICE_FISSION
 
+#if defined(CL_NAME_VERSION_MAX_NAME_SIZE_KHR)  // cl_khr_extended_versioning
```

Comment:
I'd prefer if we tested for `cl_khr_extended_versioning` but I realise I forgot to add it. I've opened https://github.com/KhronosGroup/OpenCL-Headers/pull/85 to fix this.

Reply:
I switched to the `cl_khr_extended_versioning` define after merging https://github.com/KhronosGroup/OpenCL-Headers/pull/85 - thanks!

Reply:
Thanks!